backend/graminsta/core/models.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: UTF-8 -*-
+
+"""models.py
+"""
+
+from django.db import models
+from django.contrib.auth import models as auth_models
+
+
+# Create your models here.
+class UserInfo(models.Model):
+    """UserInfo Class
+    """
+    user = models.OneToOneField(auth_models.User, on_delete=models.CASCADE)
+    age = models.IntegerField()
+    gender = models.CharField(max_length=10)
```

Comment:
For enumerable fields, use `choices` instead.

See also: https://code.djangoproject.com/ticket/18062